<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>663</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Добавить товар</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Удалить товар из базы</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Реализовать товар</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Вывести данные о товарах</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Изменить данные товара</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Соединиться с БД</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="8">
     <widget class="QTableView" name="tableView">
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_7">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Отчет о продажах</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>Выйти из программы</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>pushButton_7</zorder>
   <zorder>tableView</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_8</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
